src/app/core/components/top-bar/top-bar.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Input } from '@angular/core';
+import { Observable } from 'rxjs';
+import { ContentInfo } from '../../../shared/models/content/content-info';
+import { delay } from 'rxjs/operators';
+import { CurrentContentService } from '../../../shared/services/current-content.service';
+import { Mode, ModeAction, ModeService } from '../../../shared/services/mode.service';
+
+@Component({
+  selector: 'alg-top-bar',
+  templateUrl: './top-bar.component.html',
+  styleUrls: [ './top-bar.component.scss' ],
+})
+export class TopBarComponent {
+  @Input() scrolled = false;
+  @Input() currentMode?: Mode;
+  @Input() fullFrameContent?: boolean;
+
+  currentContent$: Observable<ContentInfo|null> = this.currentContent.content$.pipe(delay(0));
```

Comment:
I know we use the `.pipe(delay(0))` trick in other parts of the platform, but sometimes it is useless.
Can you test if removing `delay(0)` changes anything ?

Reply:
Fixed